tests/unit/visual/tst_picker.11.qml

```diff
@@ -288,10 +289,10 @@ Item {
             waitPickerScrolling();
 
             compare(linearList.count, 14, "bad removal from linearList")
-            compare(linearDynPicker.selectedIndex, 4, "bad index of linearList");
+            compare(linearDynPicker.selectedIndex, 5, "bad index of linearList");
```

Comment:
Why this is changed.

Reply:
I guess the `Object` issues affect this. Reverted now.